openeogeotrellis/backend.py

```diff
@@ -1166,8 +1166,11 @@ def get_missing_sentinel1_band() -> Optional[str]:
             summary = str_truncate(summary, width=width)
         else:
             is_client_error = False  # Give user the benefit of doubt.
+            import urllib
```

Comment:
Is there a particular reason why you're doing a function-level `import` rather than a top-level `import`?

Reply:
I thought it would save memory, but it it would be a few kb tops.